templates/pages/setup/mailcollector/collect_now.html.twig

```diff
@@ -0,0 +1,42 @@
+{#
+ # ---------------------------------------------------------------------
+ #
+ # GLPI - Gestionnaire Libre de Parc Informatique
+ #
+ # http://glpi-project.org
+ #
+ # @copyright 2015-2023 Teclib' and contributors.
+ # @copyright 2003-2014 by the INDEPNET Development Team.
+ # @licence   https://www.gnu.org/licenses/gpl-3.0.html
+ #
+ # ---------------------------------------------------------------------
+ #
+ # LICENSE
+ #
+ # This file is part of GLPI.
+ #
+ # This program is free software: you can redistribute it and/or modify
+ # it under the terms of the GNU General Public License as published by
+ # the Free Software Foundation, either version 3 of the License, or
+ # (at your option) any later version.
+ #
+ # This program is distributed in the hope that it will be useful,
+ # but WITHOUT ANY WARRANTY; without even the implied warranty of
+ # MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ # GNU General Public License for more details.
+ #
+ # You should have received a copy of the GNU General Public License
+ # along with this program.  If not, see <https://www.gnu.org/licenses/>.
+ #
+ # ---------------------------------------------------------------------
+ #}
+
+{% import 'components/form/basic_inputs_macros.html.twig' as inputs %}
+
+<div class="center mt-4">
+   <form name="form" method="post" action="{{ path('/front/mailcollector.form.php') }}">
+      {{ inputs.hidden('_glpi_csrf_token', csrf_token()) }}
+      {{ inputs.hidden('id', id) }}
+      {{ inputs.submit('get_mails', _x('button', 'Get email tickets now'), 1) }}
```

Comment:
Not sure how easy it is to move this to main form. 
We have a tab named "Actions" (plural) with only one action and nothing displayed in this tab apart the button.